migration-engine/migration-engine-tests/tests/migrations/id.rs

```diff
@@ -394,7 +399,8 @@ fn making_an_autoincrement_default_an_expression_then_autoincrement_again_works(
     });
 }
 
-#[test_connector(exclude(CockroachDb))]
+// Ignoring TiDB, we can't drop column id with composite index covered or Primary Key covered now.
```

Comment:
Is there a link for this limitation?

Reply:
Also because `unsupported drop primary key when the table is using clustered index`

update the comment on https://github.com/prisma/prisma-engines/pull/3308/commits/52d7328031bd4f31886b78244bfd2f801ccd708c